package.json

```diff
@@ -29,8 +29,8 @@
   },
   "dependencies": {
     "@paraswap/sdk": "^6.6.0",
-    "@protocolink/common": "^0.3.11",
-    "@protocolink/core": "^0.4.13",
+    "@protocolink/common": "^0.4.1",
```

Comment:
please fix the unit test issue caused by this new version of common pkg.

Reply:
fixed

Reply:
https://github.com/dinngo/protocolink-logics/pull/69/checks

Reply:
fixed the configs